src/components/stepforms/WidgetSortColumn.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <fieldset class="widget-sort__container">
+    <WidgetAutocomplete
+      id="columnInput"
+      :options="columnNames"
+      v-model="sort.column"
+      name="Column"
+      placeholder="Enter a column"
+      @input="setSelectedColumns({ column: sort.column })"
+    ></WidgetAutocomplete>
+    <WidgetAutocomplete
+      id="sortOrderInput"
+      v-model="sort.order"
+      name="Order"
+      :options="sortOrder"
+      placeholder="Order by"
+    ></WidgetAutocomplete>
+  </fieldset>
+</template>
+<script lang="ts">
+import _ from 'lodash';
+import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
+import { Getter, Mutation } from 'vuex-class';
+import { MutationCallbacks } from '@/store/mutations';
+import WidgetAutocomplete from './WidgetAutocomplete.vue';
+import { SortColumnType } from '@/lib/steps';
+
+const defaultValues: SortColumnType = {
+  column: '',
+  order: 'asc',
+};
+
+@Component({
+  name: 'widget-sort-column',
+  components: {
+    WidgetAutocomplete,
+  },
+})
+export default class WidgetSortColumn extends Vue {
+  @Prop({
+    type: Object,
+  })
+  value!: SortColumnType;
+
+  @Getter columnNames!: string[];
+
+  @Mutation setSelectedColumns!: MutationCallbacks['setSelectedColumns'];
+
+  sort: SortColumnType = { ...this.value };
+  sortOrder: SortColumnType['order'][] = ['asc', 'desc'];
```

Comment:
I would not use a specific data attribute for that because I don't see the point of being able to modify it. I would vote for just hardcoding `['asc', 'desc']` in `WidgetAutocomplete` instantiation